analysis/getEfficienciesLbL.cpp

```diff
@@ -14,7 +14,11 @@ string configPath = "configs/efficiencies.md";
 string outputPath = "results/efficienciesLbL.root";
 
 vector<string> histParams = {
-  "reco_id_eff", "trigger_eff", "trigger_single_eff", "trigger_double_eff", "charged_excl_eff", "neutral_excl_eff"
+  "reco_eff", "id_eff", "trigger_eff", "trigger_single_eff", "trigger_double_eff", "charged_excl_eff", "neutral_excl_eff"
+};
+
+vector<string> histParams_eta = {
```

Comment:
Just to be consistent, please use camel case for everything that's not a string value, i.e. histParams_eta → histParamsEta